checker/tests/regex/Issue3281.java

```diff
@@ -44,6 +44,7 @@ void m3(String s) {
     void m4(String s, String s2) {
         RegexUtil.isRegex(s);
         if (RegexUtil.isRegex(s2)) {
+            // :: error: (argument.type.incompatible)
             Pattern.compile(s);
         }
     }
```

Comment:
The PR adds the empty file `dataflow/src/main/java/org/checkerframework/dataflow/cfg/CFGBuilder.java` which should be undone.